debian/kano-settings.install

```diff
@@ -21,6 +21,7 @@ etc/rc.network /etc
 etc/rc.overclock /etc
 etc/rc.others /etc
 etc/init.d /etc
+etc/polkit-1 /etc
```

Comment:
@radujipa I think we need to copy the file explicitly, because this directory belongs to a different package, and apt would mistakenly try to clean it up on removal, as in here:

 * https://github.com/KanoComputing/os-dashboard/blob/master/debian/kano-dashboard-common.install#L14